docs/troubleshooting.rst

```diff
@@ -1,16 +1,24 @@
+.. _troubleshooting:
+
+
 Troubleshooting
 =================
 
 
 Using Relative File Paths
 ---------------------------------------------------------------
 
-OIF assumes explicit file paths so ../this_directory will cause errors. It is best to give the full output and input paths for directories and input file/ouput file locations.
+We recommend using explicit file paths so ../this_directory will cause errors. It is best to give the full 
```

Comment:
I would word this as "We recommend using explicit file paths **as ../this_directory may** cause errors". It shouldn't as long as the user is sensible and runs the command in the same directory they've defined the paths relative to, but of course users are often not sensible (and I include myself in that).